app/feature/feature-profile/src/main/kotlin/com/hedvig/android/feature/profile/tab/ProfileDestination.kt

```diff
@@ -214,6 +218,73 @@ private fun ProfileScreen(
   }
 }
 
+@Composable
+private fun ColumnScope.ProfileRowsWithPlaceholders(
```

Comment:
The `ColumnScope` here doesn't do something since it only has one composable top-level anyway, the AnimatedContent, so it can be removed.